packages/dds/tree/src/test/feature-libraries/objectForest.spec.ts

```diff
@@ -9,318 +9,34 @@ import { strict as assert } from "assert";
-            assert.equal(forest.tryMoveCursorToNode(childAnchor1, cursor), TreeNavigationResult.Ok);
-            assert.equal(cursor.value, 1);
-            assert.equal(forest.tryMoveCursorToNode(childAnchor2, cursor), TreeNavigationResult.Ok);
-            assert.equal(cursor.value, 2);
-
-            // Cleanup is not required for this test (since anchor set will go out of scope here),
-            // But make sure it works:
-            forest.anchors.forget(parentAnchor);
-            forest.anchors.forget(childAnchor1);
-            forest.anchors.forget(childAnchor2);
-            forest.anchors.forget(parentAnchor2);
-            assert(forest.anchors.isEmpty());
-        });
-
-        // TODO: test more kinds of deltas, including moves.
-
-        describe("top level invalidation", () => {
-            it("data editing", () => {
-                const forest = factory(new InMemoryStoredSchemaRepository(defaultSchemaPolicy));
-                const dependent = new MockDependent("dependent");
-                recordDependency(dependent, forest);
-
-                const content: JsonableTree[] = [{ type: jsonNumber.name, value: 1 }];
-                const insert: Delta.Insert = {
-                    type: Delta.MarkType.Insert,
-                    content: content.map(singleTextCursor),
-                };
-                // TODO: make type-safe
-                const delta: Delta.Root = new Map([[rootFieldKeySymbol, [insert]]]);
-
-                assert.deepEqual(dependent.tokens, []);
-                forest.applyDelta(delta);
-                assert.deepEqual(dependent.tokens.length, 1);
-
-                forest.applyDelta(delta);
-                assert.deepEqual(dependent.tokens.length, 2);
-
-                // TODO: maybe test some other deltas.
-            });
-
-            it("schema editing", () => {
-                const forest = factory(new InMemoryStoredSchemaRepository(defaultSchemaPolicy));
-                const dependent = new MockDependent("dependent");
-                recordDependency(dependent, forest);
-                forest.schema.update(jsonSchemaData);
-
-                assert.deepEqual(dependent.tokens.length, 1);
-            });
-        });
-    });
-
-    testJsonableTreeCursor(
-        "object-forest cursor",
-        (data): ITreeCursor => {
-            const forest = factory(
-                new InMemoryStoredSchemaRepository(defaultSchemaPolicy, jsonSchemaData),
-            );
-            initializeForest(forest, [singleTextCursor(data)]);
-            const cursor = forest.allocateCursor();
-            moveToDetachedField(forest, cursor);
-            assert(cursor.firstNode());
-            return cursor;
-        },
-        jsonableTreeFromCursor,
-        false,
-    );
-
-    // TODO: implement and test fine grained invalidation.
-}
+import { initializeForest, moveToDetachedField } from "../../forest";
+import { ITreeCursor } from "../../tree";
```

Comment:
We're likely to move the core modules to `/core` so in preparation for that, it's best to make new imports from "../../core"

Reply:
Same thing for the imports in `forestTestSuite.ts`